build.gradle

```diff
@@ -98,6 +101,11 @@ dependencies {
     testImplementation "org.junit.jupiter:junit-jupiter-engine"
     testImplementation "org.hamcrest:hamcrest"
     testImplementation "org.hamcrest:hamcrest-library"
+
+    //test container
+    testImplementation 'org.testcontainers:testcontainers:1.20.+'
```

Comment:
We usually fix the latest version and let dependabot updating it.
We never use non-exact version.